doc/architectural/central-data-structures.md

```diff
@@ -111,7 +111,7 @@ A  goto program  is a sequence of GOTO instructions (`goto_instructiont`). For
 details of the `goto_programt` class itself see the documentation in
 [`goto_program.h`](../../src/goto-programs/goto_program.h). For further details
 of the life cycle of goto programs see
-[`src/goto-programs/README.md`](../../src/goto-programs/README.md).
+[`src/goto-programs/README.md`](https://github.com/diffblue/cbmc/blob/develop/src/goto-programs/README.md).
```

Comment:
What will this be rendered as on diffblue.github.io?!

Reply:
This link follows the same format as is found here - https://github.com/diffblue/cbmc/blob/3f578e18a897ac4f47ba7cff8b0d9c04c9bb799c/doc/architectural/compilation-and-development.md?plain=1#L17
this example is rendered to https://diffblue.github.io/cbmc/compilation-and-development.html

The original link I used was causing the check-doxygen job to fail with the error - `/home/runner/work/cbmc/cbmc/doc/architectural/central-data-structures.md:120: warning: unexpected token TK_EOF as the argument of ref`

Reply:
Well, but `COMPILING.md` isn't actually rendered on diffblue.github.io, while `goto-programs/README.md` is (the latter as https://diffblue.github.io/cbmc/group__goto-programs.html). It would be great to find a way so that people aren't sent back&forth between Doxygen-rendered content and GitHub rendered pages.